tests/ci/docker_images_check.py

```diff
@@ -113,7 +119,7 @@ def get_changed_docker_images(
     for dockerfile_dir, image_description in images_dict.items():
         for f in files_changed:
             if f.startswith(dockerfile_dir):
-                name = image_description["name"]
+                name = DOCKER_REPO + "/" + image_description["name"]
```

Comment:
f-string looks more consistent here and below

Reply:
besides that, did you launch https://github.com/ClickHouse/ClickHouse/blob/master/.github/workflows/pull_request.yml#L48-L50 locally?

Reply:
I have updated it to an f-string.